vyper/compiler/output.py

```diff
@@ -84,6 +84,16 @@ def build_interface_output(compiler_data: CompilerData) -> str:
     return out
 
 
+def build_bb_output(compiler_data: CompilerData) -> IRnode:
+    compiler_data.experimental_codegen = True
```

Comment:
i don't think we should enable the flag as it introduces a kind of state into the compiler passes. so like `vyper -f bb,asm` will implicitly return asm generated through the experimental pipeline